Packs/Infoblox/Integrations/Infoblox/Infoblox.py

```diff
@@ -330,15 +471,248 @@ def parse_demisto_exception(error: DemistoException, field_in_error: str = 'text
     return DemistoException(err_msg)
 
 
+def transform_ext_attrs(ext_attrs: str) -> list:
+    """
+    Helper function to transform the extension attributes.
+    The user supplies a string of key/value pairs separated by commas.
+
+    This function parses that string and returns a list of dictionaries with "name" and "value" keys.
+
+    Args:
+    - `ext_attrs` (`str`): The string of key/value pairs separated by commas.
+
+    Returns:
+    - `list[dict]` or `None`: A `list[dict]` representing the extension attributes.
+    Returns `None` in case there were no delimiters present.
+
+    For example:
+
+    ```python
+    >>>> transform_ext_attrs("Site=Tel-Aviv")
+    [{"Site": "Tel-Aviv"}]
+
+    >>>> transform_ext_attrs("IB Discovery Owned=EMEA,Site=Tel-Aviv")
+    [{"*IB Discovery Owned": "EMEA", "*Site": "Tel-Aviv"}]
+    ```
+    """
+
+    # In case there are no delimiters present in the input
+    if "," not in ext_attrs and "=" not in ext_attrs:
+        return []
+
+    l_ext_attrs: list[dict] = []
+
+    attributes = ext_attrs.split(",")
+
+    for ext_attr in attributes:
+        try:
+            key, value = ext_attr.split("=")
+            if key and value:
+                l_ext_attrs.append({f"*{key.strip()}": value.strip()})
+        except ValueError:
+            demisto.error(f"Unable to parse ext_attrs: {ext_attrs}")
+            continue
+
+    return l_ext_attrs
+
+
+def valid_ip(ip: str):
+    """Validate IP address format"""
+
+    try:
+        ipaddress.IPv4Address(ip)
+    except ValueError:
+        raise InvalidIPAddress(f"'{ip}' is not a valid IPv4 address")
+
+
+def valid_netmask(address: str):
+    """Validate netmask format"""
+
+    try:
+        ipaddress.ip_network(address, strict=False)
+    except ValueError:
+        raise InvalidNetmask(f"'{address}' is not a valid netmask")
+
+
+def valid_ip_range(from_ip: str, to_ip: str):
+    """Validate IP range format"""
+    try:
+        from_address = ipaddress.IPv4Address(from_ip)
+        to_address = ipaddress.IPv4Address(to_ip)
+        list(ipaddress.summarize_address_range(from_address, to_address))
+    except ValueError as err:
+        raise InvalidIPRange(f"'{from_ip}' to '{to_ip}' is not a valid IPv4 range: {err}")
```

Comment:
Were we asked to add validation to user IP input? we almost never do that

Reply:
no need to validate inputs.